### PR TITLE
New IA - Quack & Hack Reference Cheat Sheet

### PR DESCRIPTION
**Pull request by zekiel**
**What does your Instant Answer do?**
Gives quick reference info for attendees of our meetup events: meetup.com/pro/duckduckgo 

**What is the data source for your Instant Answer? (Provide a link if possible)**
N/A

**Why did you choose this data source?**
N/A

**Are there any other alternative (better) data sources?**
N/A

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
DuckDuckHackers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Only in a very meta way

**Are you having any problems? Do you need our help with anything?**
No thank you. 

**Where did you hear about DuckDuckHack? (For first time contributors)**
At work. 

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

quackhack

-----
IA Page: https://duck.co/ia/view/quackhack_cheat_sheet

### CODE REVIEW

**Comment by zekiel** [comment]
//cc @tagawa  @moollaza

File: share/goodie/cheat_sheets/json/quackhack.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+    "id": "quackhack_cheat_sheet",
+    "name": "Quack & Hack Help",
+    "description": "Reference for help during Quack & Hacks",
+    "metadata": {
+        "sourceName": "DuckDuckHack",
+        "sourceUrl": "https://duck.co/duckduckhack/"
+    },
+    "aliases":["quack and hack", "quack & hack", "quack&hack", "hack and quack", "hack & quack"],
+    "template_type": "reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/quackhack.json]: My my, using templates already ;)

File: share/goodie/cheat_sheets/json/quackhack.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            "key": "Developer Documentation"
+        }, {
+            "val": "https://duck.co/ideas",
+            "key": "Instant Answer Ideas"
+        }, {
+            "val": "https://duck.co/ia",
+            "key": "Live Instant Answers"
+        }], 
+        "Communication": [{
+            "val": "duckduckhack.slack.com",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/quackhack.json]: Nit: let's add `https://`

**Comment by tagawa** [comment]
Excellent idea!
Is it possible to make it trigger on just "quack hack help" (i.e. without the ampersand)?

**Comment by namandistro** [comment]
We need it as soon as possible.... Btw Great work @zekiel nice idea!!!!

File: share/goodie/cheat_sheets/json/quackhack.json (hunk 1 above)
Comment (zekiel) [inline share/goodie/cheat_sheets/json/quackhack.json]: :+1:

**Comment by zekiel** [comment]
Thanks all 

@moollaza  assumed you meant https for the slack link -- added
@tagawa  added "quack hack" and "hack quack" as aliases --thanks! meant to do that originally
@namandistro  awesome! Feel free to improve on it in the future if you think we should add/remove something

**Comment by rohitzidu** [comment]
@zekiel add different team name from different location contributing to quack and hack

**Comment by zekiel** [comment]
@rohitzidu  you mean organizer names? I'm not comfortable publishing personal information in an instant answer so instead added a link to the meetup groups (meetup.com/pro/duckduckgo) to Quick Links section

**Comment by rohitzidu** [comment]
No, I mean to say group name and meet up group address, if anybody looks on cheat sheet he can find near-by quack n hack meet-up group.
Give link to it so it redirect to meetup group.

Example: quack & hack delhi
Meetup.com/quack-hack-delhi

-rohit

**Comment by namandistro** [comment]
@rohitzidu && @zekiel can we have IA saying "DDH near me" and by getting their location we can recommend him the nearest DDH community What say? Can we do that?

**Comment by tagawa** [comment]
@namandistro "DDH near me" - I like that! It would depend on whether there's a geocoding service that can be used. I'd suggest [submitting it as an idea](https://duck.co/ideas) to get better feedback.

**Comment by namandistro** [comment]
@tagawa See for example, We know how many DDH community is their in India. DDG shows location in side so what we have to do is we know the hit we just have to find which community is near and suggest one or two more which are around him.

**Comment by rohitzidu** [comment]
@naman @daniel @zac ,

I think we should integrate in this cheatsheet, as it would contain all information user want to know about duckduckhack or quack & hack.
It will have everything in it:-
About instant answer , types , group & location, repo , chat communications.
Rohit

**Comment by namandistro** [comment]
@moollaza that wilk be messy and not even needed this cheat sheet just for quick help to people for devlopment.

**Comment by zekiel** [comment]
@namandistro i agree -- this is optimally for use AT a meetup event. @rohitzidu  your idea is great but should start as a separate IA until something robust/dynamic enough comes along. (similar to the big DDG answers_IA https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/DuckDuckGo.pm )

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@zekiel @tagawa @namandistro @rohitzidu FWIW there is an in-progress "Meetup.com" Spice which will ideally be able to search for Meetups near you, for a given topic. It might be the best way to handle queries like "DDH near me"

**Comment by namandistro** [comment]
‎I submitted the same Idea with meetup refrence.
From: Zaahir Moolla
Sent: Wednesday 19 August 2015 10:42 AM
To: duckduckgo/zeroclickinfo-goodies
Reply To: duckduckgo/zeroclickinfo-goodies
Cc: Naman Distro
Subject: Re: [zeroclickinfo-goodies] New IA - Quack & Hack Reference Cheat Sheet (#1395)


@zekiel @tagawa @namandistro @rohitzidu FWIW there is an in-progress "Meetup.com" Spice which will ideally be able to search for Meetups near you, for a given topic. It might be the best way to handle queries like "DDH near me"

---

**Comment by moollaza** [comment]
Congrats! It's live: https://duckduckgo.com/?q=quack+and+hack+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**